README.md

```diff
@@ -143,12 +143,17 @@ The action can be configured with the following arguments:
 
 - `plan` - (optional) Used for
   [update plans](https://www.pulumi.com/docs/concepts/update-plans/)
+
   - On `preview`: Where to save the update plan. If you choose to use this in a
     different run of your workflow (let's say you create an update plan via a
     preview on Pull Request creation and want to `up` using the plan) you must
     upload the plan as an artifact, and retrieve it wherever you run `up`
   - On `up`: Where to read the update plan from.
 
+- `trim-comments-from-front` - (optional) If `true`, then the action will trim
```

Comment:
```suggestion
- `always-include-summary` - (optional) If `true`, then the action will trim
```

Trimming from the front is what it does, but maybe this makes the intention clearer?

Reply:
Yeah, I like that.